### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by greatsamist**

### CODE REVIEW

**Comment by xannyxs** [comment]
@greatsamist can you make a quick of the things you did?

File: packages/app/app/[organization]/components/ChannelDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      }
+    }
+    handleResize()
+    window.addEventListener('resize', handleResize)
+
+    return () => window.removeEventListener('resize', handleResize)
+  }, [contentRef])
+
+  return (
+    <div className="hidden md:block w-4/5">
```
Comment (xannyxs) [inline packages/app/app/[organization]/components/ChannelDescription.tsx:29]: Guess this is on purpose, but I do not see the description at all anymore on mobile

Comment (greatsamist) [reply packages/app/app/[organization]/components/ChannelDescription.tsx:29]: Yes, I follow the previous logic of hiding the text on mobile, but we can reduce the font size if we want to have a description on mobile.

File: packages/app/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -34,6 +34,31 @@ export const SidebarUI = ({ children }: { children: ReactNode }) => {
       <SidebarContext.Provider value={{ expanded }}>
         <ul className="flex-1 px-3 text-white">{children}</ul>
       </SidebarContext.Provider>
+
```
Comment (xannyxs) [inline packages/app/components/Sidebar/Sidebar.tsx:37]: This should have been implemented differently... let me change it for a sec.

File: packages/app/components/misc/Support/index.tsx [hunk 3]
```diff
@@ -61,8 +62,8 @@ const Support = () => {
   }
   return (
     <Credenza open={open} onOpenChange={setOpen}>
-      <CredenzaContent className="!z-50">
-        <CredenzaTitle>Bug/Content Error Report</CredenzaTitle>
+      <CredenzaContent className="!z-50 overflow-auto max-h-[700px]">
```
Comment (xannyxs) [inline packages/app/components/misc/Support/index.tsx:65]: Nice job! Only thing is that it does not have padding on mobile

**Pull request merged**